### PR TITLE
fix(patternfly): adds patternfly pipe module in ngx-forge

### PR DESCRIPTION
**Pull request by invincibleJai**
- adds patternfly pipe module in ngx-forge

### CODE REVIEW

File: src/app/launcher/launcher.module.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ import { BsDropdownConfig, BsDropdownModule } from 'ngx-bootstrap/dropdown';
 import { PopoverModule } from 'ngx-bootstrap/popover';
 import { TypeaheadModule } from 'ngx-bootstrap/typeahead';
 
-import { SortArrayPipeModule, TruncatePipeModule } from 'patternfly-ng/pipe';
+// import { SortArrayPipeModule, TruncatePipeModule } from 'patternfly-ng/pipe';
```
Comment (gastaldi) [inline src/app/launcher/launcher.module.ts]: Delete this line

File: src/app/launcher/shared/pipe/truncate/truncate.pipe.spec.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { TruncatePipe } from './truncate.pipe';
+
+describe('Truncate pipe', () => {
+  // This pipe is a pure, stateless function so no need for BeforeEach
+  let pipe = new TruncatePipe();
+
+  it('should transforms "this is a not so long string" to "this is a not so lon..."', () => {
```
Comment (gastaldi) [inline src/app/launcher/shared/pipe/truncate/truncate.pipe.spec.ts]: `should transform`

**Comment by dlabrecq** [comment]
FYI, I regenerated the package-lock for ngx-forge and fabric8-planner using the latest patternfly-ng. Now that the planner PR has been merged, I'll update fabric8-ui.

**Comment by invincibleJai** [comment]
@dlabrecq Agree with you Dan. I hope this PR would not be needed as the fix you are working on should help. Have raised it as backup. As update to prod is blocked.

**Comment by dlabrecq** [comment]
Instead of duplicating patternfly-ng code, we could have reused PipeModule for the time being. The change was simply to remove the deprecation, but PipeModule is still there. 

That said, I believe that fabric8-ui is pulling an incorrect version of ngx-forge (i.e., considering the errors reference old code). Regardless of which solution we choose here, fabric8-ui must be fixed or it's not going to install the package correctly.

I've created a fabric8-ui PR here: https://github.com/fabric8-ui/fabric8-ui/pull/3069

**Comment by dlabrecq** [comment]
Tested the latest prod-preview and I was able to create an application successfully. This can be closed now.

**Pull request closed without merging**